python/agents/image-scoring/deployment/deploy.py

```
@@ -0,0 +1,129 @@
+"""Deployment script for Image Scoring."""
+
+import os
+
+from absl import app
+from absl import flags
+from dotenv import load_dotenv
+
+
+load_dotenv()
+
+from image_scoring.agent import root_agent
```

Flake8 (7.2.0)

[error] module level import not at top of file

(E402)